### ISSUE TITLE
Suggestion: Add 1.9 Enchantments to the Enchanter.

### ISSUE THREAD

**Issue by VT-14** [SleepyTrousers/EnderIO#3364]
I didn't see them in Ender IO 1.9.4-3.0.0.36_alpha.

I think there is just Mending and Frost Walker to add.  Depth Strider was added in 1.8, but I think there was a recipe for that already.

I think it would be fair for Mending to be quite expensive.  Probably 40 Levels and a Nether Star.

I don't think Frost Walker is as desirable.  Still, it's difficult to get, so maybe using Compressed Ice.

**Comment by HenryLoenwind** [comment]
I'm a bit surprised the Enchanter works at all ;).

BTW: Those enchantments come from an XML config file, so you can add them yourself. It'd be very helpful if you could find some suitable recipes, add them to your config, test them and then post your xml here. If you think those recipes would be too cheap with the existing items, we can add more crystals. So a freezing crystal made from e.g. a diamond and a Polar Bear soul (1.10, I know) would be possible.

**Comment by VT-14** [comment]
Honestly, I just saw that there weren't recipes in JEI when I made this suggestion.  I've since tested and confirmed that it is working on my relatively small pack.  I would like to say that the sound it makes is absolutely terrifying when you aren't expecting it though.  The Dark Steel Anvil has the same bug.

Here is my current user file:
https://gist.github.com/VT-14/a7958f7cb2dd1b2f4d1a71d25aa40c76

I made the following changes:
Mending: 1 Experience Rod and 30 levels. (The experience rod is probably too cheap, but it does fit the enchantment nicely.)

Frost Walker: 16 Ice and 8 levels for Frost Walker I, 32 ice and 20 levels for Frost Walker III.  (packed ice seems a little too rare).

I added back in the missing Thorns recipe as well.  I copies a previous version's config for it and only updated what was needed for the slightly changed format.
1,2,or 3 Rose Bushes for 12, 36, or 76 levels.


Here is what I think the EnchanterRecipes_Core.xml would look like with those changes.  I can't test to see if it works as it gets overwritten when EnderIO launches.  In this case, I also removed a duplicate aqua affinity entry.

https://gist.github.com/VT-14/a5772fbccfe82a3730275ce0e719d9ef
